modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa.dm

```diff
@@ -597,23 +597,35 @@
 	icon_state = "hecu_uniform"
 	inhand_icon_state = "r_suit"
 
+///It looks fairly fitting for an "elite tacticool squad", so we'll just reuse it with the Expeditionary Corps' gear stats, and without that plasteel mention, until someone sprites/finds a better one. To make it fair.
+/obj/item/clothing/suit/armor/vest/marine/hecu
+	desc = "A set of the finest mass produced, stamped steel armor plates, containing an environmental protection unit for all-condition door kicking."
+	armor = list(MELEE = 40, BULLET = 40, LASER = 40, ENERGY = 40, BOMB = 40, BIO = 0, FIRE = 80, ACID = 100, WOUND = 30)
+
+/obj/item/clothing/head/helmet/marine/hecu
+	armor = list(MELEE = 30, BULLET = 30, LASER = 30, ENERGY = 30, BOMB = 30, BIO = 0, FIRE = 80, ACID = 100, WOUND = 30)
+
+/obj/item/storage/backpack/ert/odst/hecu
+	name = "hecu backpack"
+
 /datum/outfit/hecu
 	name = "HECU Grunt"
 	uniform = /obj/item/clothing/under/rank/security/officer/hecu
-	head = /obj/item/clothing/head/helmet
+	head = /obj/item/clothing/head/helmet/marine/hecu
 	mask = /obj/item/clothing/mask/gas/hecu2
 	gloves = /obj/item/clothing/gloves/combat
-	suit = /obj/item/clothing/suit/armor/vest
+	suit = /obj/item/clothing/suit/armor/vest/marine/hecu
+	belt = /obj/item/storage/belt/security/webbing
 	shoes = /obj/item/clothing/shoes/combat
-	back = /obj/item/storage/backpack
-	backpack_contents = list(/obj/item/radio, /obj/item/ammo_box/magazine/m16 = 4, /obj/item/storage/firstaid/expeditionary)
+	back = /obj/item/storage/backpack/ert/odst/hecu
+	backpack_contents = list(/obj/item/radio, /obj/item/ammo_box/magazine/m16 = 4, /obj/item/storage/firstaid/expeditionary, /obj/item/storage/box/mothic_rations, /obj/item/gun/ballistic/automatic/pistol/g17, /obj/item/ammo_box/magazine/multi_sprite/g17 = 2, /obj/item/flashlight/glowstick = 2, /obj/item/knife/combat, /obj/item/grenade/frag, /obj/item/clothing/mask/balaclavaadjust)
```

Comment:
this is too much stuff, it doesn't all fit.

Reply:
Mothic ration is to keep them not starving until the Vanguard folks arrive, Glock is an obvious sidearm, same for the knife. Glowsticks are there for illumination purposes as there's quite a bunch of dark areas that'd use some (temporary) lights. Frag was added in due to HECU being notoriously known for throwing a lot of grenades, and balaclava, as already mentioned, is for the style in case someone decides to use it instead of the gas mask.
That ODST backpack was added in due to looking fairly generic and kinda fitting to how HECU looks, so, yes.

Reply:
Ok but it's failing checks because it does not all fit in the bag

Reply:
Oh.
I see what you mean by "it doesn't fit" now.
I legitimately didn't notice that at first.
Brb.

Reply:
Alright, that should work now.